### PR TITLE
testing/wlroots: new aport

### PR DESCRIPTION
**Pull request by HRio**
While at it upgrade main/wayland-protocols

Note this change requires https://github.com/alpinelinux/aports/pull/5591 to be commited as well

### CODE REVIEW

File: testing/wlroots/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+pkgdesc="A modular Wayland compositor library"
+url="https://github.com/swaywm/wlroots"
+arch="all"
+license="MIT"
+options="!check" # contains no test suite
+makedepends="ctags
+	eudev-dev
+	libcap-dev
+	libinput-dev
+	libpng-dev
```
Comment (emersion) [inline testing/wlroots/APKBUILD]: This shouldn't be required. You can add `-Dexamples=false` to Meson flags to make sure examples aren't built.

Comment (HRio) [reply testing/wlroots/APKBUILD]: it still checks for libpng with this diff
```diff
--- a/testing/wlroots/APKBUILD
+++ b/testing/wlroots/APKBUILD
@@ -27,7 +27,7 @@ builddir="$srcdir/$pkgname-$pkgver"

 build() {
        cd "$builddir"
-       meson build --prefix /usr
+       meson build --prefix /usr -Dexamples=false
        ninja -C build
 }
```

```log
Dependency threads found: YES
Dependency wayland-cursor found: YES 1.16.0
Dependency libpng found: YES 1.6.35
Dependency libavutil found: NO (tried pkgconfig)
Dependency libavcodec found: NO (tried pkgconfig)
Dependency libavformat found: NO (tried pkgconfig)
```

Comment (emersion) [reply testing/wlroots/APKBUILD]: It checks for dependencies, but doesn't use them. Same for `libav*` for instance.

**Comment by emersion** [comment]
Maybe you'd want the XCB dependencies for Xwayland support (`libxcb-dev`, `xcb-util-image-dev`, `xcb-util-wm-dev`, maybe others).

**Comment by HRio** [comment]
wlroots: 	d5f0369a8bc736a04b0b0963865618047caee8f9 

Will make the main/wayland-protocols a new PR.

**Comment by HRio** [comment]
main/wayland-protocols is #5764

**Pull request closed without merging**